src/Web/Shared/CreateEmployeeValidator.cs

```diff
@@ -0,0 +1,22 @@
+﻿using FluentValidation;
+
+public class CreateEmployeeValidator : AbstractValidator<CreateEmployee>
+{
+    public CreateEmployeeValidator()
+    {
+        RuleFor(s => s.Email).EmailAddress();
+        RuleFor(s=>s.Username).NotEmpty();
+        RuleFor(s=>s.FirstName).NotEmpty();
+        RuleFor(s=>s.LastName).NotEmpty();
+        RuleFor(s => s.PhoneNumber).Matches(@"^(?:\d{9}|\+992\d{9}|992\d{9})$")
```

Comment:
remove
       RuleFor(s => s.PhoneNumber).Matches(@"^(?:\d{9}|\+992\d{9}|992\d{9})$")
            .WithMessage("Invalid phone number. Example : +992921234567, 992921234567, 921234567");
            
            This is client validation